src/main/java/nf_core/nf/test/utils/Methods.java

```diff
@@ -0,0 +1,35 @@
+package nf_core.nf.test.utils;
+
+import org.yaml.snakeyaml.Yaml;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Map;
+
+public class Methods {
+  public static Map<String, Map<String, Object>> readYamlFile(String filePath) {
+    Yaml yaml = new Yaml();
+
+    System.err.println(filePath);
```

Comment:
```suggestion
```

Reply:
yeah I was thinking the same, but that's a good first start already